fastlane/jetpack_metadata/zh-Hans/subtitle.txt

```diff
@@ -1 +1 @@
-提高您 WordPress 站点的速度和安全性。
\ No newline at end of file
+更快速、更安全的 WordPress
```

Comment:
18 characters ✅

That is if App Store Connect does something like "...".length, which would make sense to me given that the 30 characters limitation is not a matter of sentence complexity but of screen real estate. I'm not actually sure if Chinese characters are counted in the same way as we do with the letters in a Roman language.